templates/cisco_nxos_show_ip_route.textfsm

```diff
@@ -15,10 +15,14 @@ Value TUNNELID (0x[a-f\d]+)
 Value ENCAP (\w+)
 
 Start
+  ^.+show\sip\sroute
```

Comment:
Do you mind removing this line and the line this accounts for in the output as well?

We typically don't have the command within the output.

Reply:
Removed the command from the .raw and the match